sdk/resource/process_test.go

```diff
@@ -107,15 +107,15 @@ func restoreAttributesProviders() {
 func TestWithProcessFuncs(t *testing.T) {
 	mockProcessAttributesProviders()
 
-	t.Run("WithPID", testWithProcessPID)
-	t.Run("WithExecutableName", testWithProcessExecutableName)
-	t.Run("WithExecutablePath", testWithProcessExecutablePath)
-	t.Run("WithCommandArgs", testWithProcessCommandArgs)
-	t.Run("WithOwner", testWithProcessOwner)
-	t.Run("WithRuntimeName", testWithProcessRuntimeName)
-	t.Run("WithRuntimeVersion", testWithProcessRuntimeVersion)
-	t.Run("WithRuntimeDescription", testWithProcessRuntimeDescription)
-	t.Run("WithProcess", testWithProcess)
+	t.Run("WithPID", TestWithProcessPID)
+	t.Run("WithExecutableName", TestWithProcessExecutableName)
+	t.Run("WithExecutablePath", TestWithProcessExecutablePath)
+	t.Run("WithCommandArgs", TestWithProcessCommandArgs)
+	t.Run("WithOwner", TestWithProcessOwner)
+	t.Run("WithRuntimeName", TestWithProcessRuntimeName)
+	t.Run("WithRuntimeVersion", TestWithProcessRuntimeVersion)
+	t.Run("WithRuntimeDescription", TestWithProcessRuntimeDescription)
+	t.Run("WithProcess", TestWithProcess)
```

Comment:
This runs these tests twice in the CI system, right? Can we remove this and keep the exported functions.

Reply:
Already done as mentioned above, please review it again. Thanks